runtime/jcl/common/system.c

```diff
@@ -161,7 +163,7 @@ Java_java_lang_System_getSysPropBeforePropertiesInitialized(JNIEnv *env, jclass
 #if defined(J9ZOS390) || defined(J9ZTPF)
 			sysPropValue = "ISO8859_1";
```

Comment:
Please change this to `"ISO-8859-1"`.

Reply:
`OpenJ9` uses `ISO8859_1` across codebase, I think it worths a separated PR for such refactoring.

Reply:
Can you add a link to that issue here, please?

Reply:
Created https://github.com/eclipse-openj9/openj9/issues/14301 to track this.